node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/DedicatedClusterControllerClusterMigrator.java

```diff
@@ -44,6 +45,7 @@ protected DedicatedClusterControllerClusterMigrator(Deployer deployer, Metric me
 
         ZonedDateTime date = ZonedDateTime.ofInstant(clock().instant(), java.time.ZoneId.of("Europe/Oslo"));
         if (   ! nodeRepository().zone().system().isCd()
+            &&   nodeRepository().zone().environment() != Environment.staging
```

Comment:
Since we wait 10 minutes after last deployment with migration I think most deployments will not migrate anyway, but I guess this is OK.

Reply:
Yes, most deployments won't. We could also lower that, I guess, since it's just for the purpose of a working integration test.

Reply:
Anything special about staging or should this be `!nodeRepository().zone().environment().isTest()`?